src/httpurr/client/node.cljs

```diff
@@ -48,7 +49,9 @@
             (on-abort [err]
               (callback (HttpResponseError. :abort nil)))
             (on-response [msg]
-              (listen msg "readable" (partial on-message msg)))
+              (let [chunks (atom [])]
```

Comment:
There is probably a better way of doing this. I mostly followed the advice in the nodejs docs (https://nodejs.org/api/http.html#http_event_response)

> If no 'response' handler is added, then the response will be entirely discarded. However, if a 'response' event handler is added, then the data from the response object must be consumed, either by calling response.read() whenever there is a 'readable' event, or by adding a 'data' handler, or by calling the .resume() method. Until the data is consumed, the 'end' event will not fire. Also, until the data is read it will consume memory that can eventually lead to a 'process out of memory' error.